packages/patrol_cli/lib/src/features/run_commons/device.dart

```diff
@@ -25,6 +25,28 @@ class Device {
         return name;
     }
   }
+
+  String get description {
+    switch (targetPlatform) {
+      case TargetPlatform.android:
+        if (platformDescription.isEmpty) {
+          return id;
+        } else {
+          return '$platformDescription $id';
+        }
+      case TargetPlatform.iOS:
+        return '$platformDescription $name';
+    }
+  }
+
+  String get platformDescription {
+    switch (targetPlatform) {
+      case TargetPlatform.android:
+        return real ? 'device' : '';
```

Comment:
It's an empty string to avoid having platformDescription like `emulator emulator-5554`. emulator-555x is the conventional AVD id.

Bad example:
```
apk with entrypoint notifications_test.dart on emulator emulator-5554
```

Good example:
```
apk with entrypoint notifications_test.dart on emulator-5554
```